dodo.py

```diff
@@ -200,25 +200,6 @@ def task_test():
         ],
     }
 
-def task_login():
-    '''
-    perform ECR docker login via AWS perms
-    '''
-
-    def login():
-        ENVS = envs(
-            AWS_REGION=CFG.AWS_REGION,
-            AWS_ACCOUNT=CFG.AWS_ACCOUNT,
-        )
-        cmd = f'env {ENVS} aws ecr get-login-password --region {CFG.AWS_REGION} | env {ENVS} docker login --username AWS --password-stdin {CFG.ECR_REPOURL}'
-        call(cmd)
-    return {
-        'actions': [
-            login,
-        ],
-        'uptodate': [lambda: CFG.IS_AUTHORIZED],
```

Comment:
If you delete this task, you can delete `IS_AUTHORIZED` as well – it isn't used anywhere else.

There is some documentation mentioning the `login` task in `SRE_INFO.md` and `docs/refractr-architecture.md`, which you could update as well.

(I wasn't sure whether we actually need to delete this task, or whether we should keep it just in case, but I came to the conclusion that it's always better to authenticate to AWS by some external means rather than calling `doit` for it.)